### PR TITLE
Bevy 0.10 support

### PR DESCRIPTION
**Pull request by BrandonDyer64**
Just a 1:1 update to Bevy 0.10

All of the examples appear to run normally

### CODE REVIEW

File: src/pipeline.rs [hunk 1]
```diff
@@ -139,24 +139,18 @@ impl Plugin for AtmospherePipelinePlugin {
...
-            .add_system_to_stage(
-                RenderStage::Prepare,
+            .add_system(extract_atmosphere_resources.in_schedule(ExtractSchedule))
+            .add_system(Events::<AtmosphereUpdateEvent>::update_system.in_set(RenderSet::Prepare))
+            .add_system(
                 prepare_atmosphere_assets
-                    .label(PrepareAssetLabel::PostAssetPrepare)
-                    .after(PrepareAssetLabel::AssetPrepare),
+                    .in_set(PrepareAssetSet::PostAssetPrepare)
+                    .after(PrepareAssetSet::AssetPrepare),
```
Comment (BrandonDyer64) [inline src/pipeline.rs:147]: I don't think this `.after()` is actually necessary

File: src/plugin.rs [hunk 2]
```diff
@@ -137,9 +134,9 @@ fn atmosphere_remove(
     mut commands: Commands,
     parents: Query<&Children>,
     atmosphere_skyboxes: Query<Entity, With<AtmosphereSkyBox>>,
-    atmosphere_cameras: RemovedComponents<AtmosphereCamera>,
+    mut atmosphere_cameras: RemovedComponents<AtmosphereCamera>,
 ) {
-    for camera in &atmosphere_cameras {
+    for camera in atmosphere_cameras.iter() {
```
Comment (Shatur) [inline src/plugin.rs:139]:
```suggestion
    for camera in &mut atmosphere_cameras {
```

**Review by Shatur — CHANGES_REQUESTED** [review]

**Review by Shatur — APPROVED** [review]

**Comment by Shatur** [comment]
@BrandonDyer64 could you cleanup `deny.toml`? The warning are outlined in https://github.com/JonahPlusPlus/bevy_atmosphere/actions/runs/4392830639/jobs/7692841809.2

**Comment by BrandonDyer64** [comment]
Yeah I'll have a look

**Comment by BrandonDyer64** [comment]
Would you prefer adding an exception to the deny file, or switching xml-rs to another crate, like quick-xml?

**Comment by Shatur** [comment]
> Would you prefer adding an exception to the deny file, or switching xml-rs to another crate, like quick-xml?

I would add is as exception since it's used by the dependency.

**Comment by JonahPlusPlus** [comment]
Thank you for your work here. I had already started a 0.6 branch (#42), but I suppose if people really want a quick replacement we could make this the 0.6 version and release my update as 0.7.

**Comment by BrandonDyer64** [comment]
@JonahPlusPlus I like the idea of getting something in quick for Bevy 0.10 support, and having another update with all the goodies you've got in store for us at a later date. Many devs, like myself are eager to start working with the engine update, and would love to include bevy_atmosphere in their projects as soon as possible.

**Review by Shatur — APPROVED** [review]

**Comment by JonahPlusPlus** [comment]
Okay, I'll review and merge this, make some minor changes and it should be released later today or tomorrow.

File: src/pipeline.rs (hunk 1 above)
Comment (JonahPlusPlus) [reply src/pipeline.rs:147]: Yes, probably a sanity check. I'll remove it.

**Review by JonahPlusPlus — APPROVED** [review]
Looks good to me!

**Pull request merged**